### ISSUE TITLE
Sometimes can not grab and move

### ISSUE THREAD

**Issue by kohei1218** [SCENEE/FloatingPanel#320]
### Short description
I added the FloatingPanel in the FloatingPanel's ContentView.
In this case, I sometimes can not grab and move the FloatinPanel.
It works in a tracked TableView.
### Expected behavior
It should be able to move with GrabberView and Header part.
### Steps to reproduce

```
class FloatingContentViewController: UIViewController {

     override func viewDidLoad() {
        super.viewDidLoad()

        fpc = FloatingPanelController()

        let contentVC = SecondContentViewController()
        fpc.set(contentViewController: contentVC)

        fpc.track(scrollView: contentVC.tableView)

        fpc.addPanel(toParent: self)
       }
}
```

### Environment
FloatinPanel 1.7.2
Xcode 11.2.1
Mac OS 10.15.1 Beta（19B77a)

**Installation method**
CocoaPods

I attach the URL of the video.
https://imgur.com/a/ouRC62n

**Comment by SCENEE** [comment]
I'm wondering why the grabber handle is above the white surface. The panel's pan gesture is only active on the `FloatingPanelController.surfaceView`. Do you modify `contentInsets`, `containerMargin' or grabber handle position of the `FloatingPanelSurfaceView`?

**Comment by kohei1218** [comment]
I'm sorry, I forgot to describe, but I'm setting ContanerMargins.

```
surfaceView.containerMargins = .init(top: 24.0, left: 0.0, bottom: view.safeAreaInsets.bottom, right: 0.0)
```

**Comment by SCENEE** [comment]
Ok, I got it. Then I tried to reproduce this in the Maps sample app with below patch, but it's working well. So I think other conditions causes this issue 🤔 

```diff
diff --git a/Examples/Maps/Maps/ViewController.swift b/Examples/Maps/Maps/ViewController.swift
index 73ea5e1..c8c0a9d 100644
--- a/Examples/Maps/Maps/ViewController.swift
+++ b/Examples/Maps/Maps/ViewController.swift
@@ -27,6 +27,7 @@ class ViewController: UIViewController, MKMapViewDelegate, UISearchBarDelegate,
             fpc.surfaceView.cornerRadius = 0.0
         }
         fpc.surfaceView.shadowHidden = false
+        fpc.surfaceView.containerMargins = UIEdgeInsets(top: 44.0, left: 0.0, bottom: 0.0, right: 0.0)
 
         searchVC = storyboard?.instantiateViewController(withIdentifier: "SearchPanel") as? SearchPanelViewController
 
```

**Comment by kohei1218** [comment]
I created a repository where the problem happens.
It happens from time to time.
I would be grateful if you could point out any implementation issues🙇‍♂️

https://github.com/kohei1218/FloatingPanelSample

**Comment by SCENEE** [comment]
@kohei1218 Thank you so much for your work. That's helpful and finally I acknowledged this issue. I appreciate it.

I took a look at the problem, it looked that the pan gesture recognizer of the second panel sometimes doesn't work well because of both of the failure requirement and the captured are.

This should be same as #308. I will try to fix it.

**Comment by SCENEE** [comment]
I confirm this issue is fixed on `fix-2nd-panel-pan-gesture` branch! Could you try it?

**Comment by kohei1218** [comment]
It works great!!🙏
Thanks for the wonderful library.
When will this fix be released?

**Comment by SCENEE** [comment]
Glad to hear that 🎉  I've added a sample code based on your nice sample for this case.
So I'm going to release it until next weekend 👍

**Comment by SCENEE** [comment]
I've released v1.7.3 now. So I close this issue. Thank you for your contribution 👍

**Comment by kohei1218** [comment]
Thank you so much for the quick fix!!!!!
And thanks for the wonderful library👍
